doc/source/mapdl_commands/apdl/abbreviations.rst

```diff
@@ -1,22 +1,21 @@
-.. _ref_abbreviations_commands_api:
 
-*************
-Abbreviations
-*************
+.. _ref_abbreviations:
+
 
-.. currentmodule:: ansys.mapdl.core
+Abbreviations
+=============
 
-These APDL commands can be used to define abbreviations for longer commands, and to create user-defined commands.
 
-.. note::
-   It is probably easier to use Python strings instead of MAPDL
-   strings.
+.. currentmodule:: ansys.mapdl.core._commands.apdl.abbreviations
 
+.. autoclass:: ansys.mapdl.core._commands.apdl.abbreviations.Abbreviations
 
 .. autosummary::
-   :toctree: _autosummary/
+   :template: base.rst
+   :toctree: _autosummary
+
 
-   Mapdl.abbr
-   Mapdl.abbres
-   Mapdl.abbsav
-   Mapdl.ucmd
+   Abbreviations.ucmd
+   Abbreviations.abbres
+   Abbreviations.abbr
+   Abbreviations.abbsav
```

Comment:
Ok... So... if there are commands like `input` which is "extended" (we change the behaviour to adapt to the diferent interfaces, or to adapt to python), we should probably not point to the docstring in the original code.

For instance, if we change the command ``ABBRES`` to do something inside the function that might change the docstring, we should not point the documentation to ``Abbreviations.abbres`` rather ``_MapdlCommandExtended.abbres`` (assuming the new ``ABBRES`` is coded in the ``_MapdlCommandExtended`` class.

I think this is also related to: https://github.com/ansys/pymapdl/pull/3385#discussion_r1862518939

But I think for the moment, we should not worry much, since most of the commands are not going to be expanded. Just a few, for instance ``input`` which changes because of the gRPC interface. In that case, we might have both:
- the original MAPDL command in ``run_controls.py``
- the extended MAPDL command in ``mapdl_grpc.py``, and this one will overwrite the docstring of the above.

What will happen when we search for input?... presumably two results will come up...

Reply:
TLDR: Dont bother at the moment. We will figure out this in the future.